backend/cmd/api/env.sample

```diff
@@ -29,3 +29,9 @@ AZ_SIGNING_KEY=<BASE 64 ENCODED SIMMETRYC KEY>
 
 # Other Secrets
 SERVICE_SECRET_DIRECTUS=<COMMON SECRET>
+
+# Redis
+REDIS_ADDRESS=
+REDIS_USERNAME=
+REDIS_PASSWORD=
```

Comment:
Is localhost default? If not you could add localhost to address so it works with compose out of the box.